zaza/openstack/charm_tests/glance/tests.py

```diff
@@ -222,3 +223,51 @@ def test_100_create_delete_image(self):
         )
         self.assertEqual(image["size"], response["ContentLength"])
         openstack_utils.delete_image(self.glance_client, image["id"])
+
+
+class GlanceCinderBackendTest(test_utils.OpenStackBaseTest):
+    """Encapsulate glance tests using an external S3 backend."""
```

Comment:
"external S3 backend" should be "cinder backend"?

Reply:
Fixed in the 3rd commit